File: MAP/Sem1_223/src/model/Task.java
package model;

public abstract class Task {
    private String taskId;
    private String descriere;

    public Task(String taskId, String descriere) {
        this.taskId = taskId;
        this.descriere = descriere;
    }

    public String getTaskId() {
        return taskId;
    }

    public void setTaskId(String taskId) {
        this.taskId = taskId;
    }

    public String getDescriere() {
        return descriere;
    }

    public void setDescriere(String descriere) {
        this.descriere = descriere;
    }

    @Override
    public String toString() {
        return
                "id=" + taskId + '|' +
                "descriere=" + descriere;
    }

    public abstract void execute();
}

File: MAP/Seminar10_11/Sem10_MAP_223/Sem10_MAP_223/model/Task.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sem10_MAP_223.model
{
     abstract class Task : IHasID<string>
    {
        public string ID { get ; set; }

        private string description;

        public string Description
        {
            get { return description; }
            set { description = value; }
        }

        public abstract void Execute();

        public override string ToString()
        {
            return ID +" "+Description;
        }

       
    }
}
